src/views/Dashboard.vue

```diff
@@ -55,6 +56,7 @@ import { generateUrl, imagePath } from '@nextcloud/router'
 import { DashboardWidget, DashboardWidgetItem } from '@nextcloud/vue-dashboard'
 import orderBy from 'lodash/fp/orderBy'
 import prop from 'lodash/fp/prop'
+import EmptyContent from '@nextcloud/vue/dist/components/EmptyContent'
```

Comment:
it says the same also for import EmptyContent from '@nextcloud/vue/dist/Components/EmptyContent'. Whats the right path?

Reply:
Ah mail is still on version 1.5.0 but EmptyContent is only part of 2.0.0+

Reply:
Then we need https://github.com/nextcloud/mail/pull/3466

Reply:
done